.github/workflows/ci.yaml

```diff
@@ -280,11 +303,35 @@ jobs:
 
       - name: Set up JDK ${{ env.INT_TEST_JAVA_RUNTIME_VERSION }} (${{ env.INT_TEST_JAVA_RUNTIME_VENDOR }}) to run integration test
         uses: actions/setup-java@v4
-        if: ${{ !(env.INT_TEST_JAVA_RUNTIME_VERSION == env.JAVA_VERSION && env.INT_TEST_JAVA_RUNTIME_VENDOR == env.JAVA_VENDOR) }}
+        if: ${{ env.SET_UP_ORACLE_JDK == 'false'}}
         with:
           java-version: ${{ env.INT_TEST_JAVA_RUNTIME_VERSION }}
           distribution: ${{ env.INT_TEST_JAVA_RUNTIME_VENDOR }}
```

Comment:
For CosmosDB, I keep using the `actions/setup-java@v4` to setup the integration test JDK when it's not Oracke JDK 8 and 11 that is demanded.  This is because we need to install TLS/SSL certificate into the Java installation before `gradle integrationTestCosmos` is run.